be/src/olap/tablet.cpp

```diff
@@ -1999,8 +1999,8 @@ Status Tablet::calc_delete_bitmap(RowsetId rowset_id,
                 RowLocation loc;
                 // first check if exist in pre segment
                 if (check_pre_segments) {
-                    auto st = _check_pk_in_pre_segments(pre_segments, *key, dummy_version,
-                                                        delete_bitmap, &loc);
+                    auto st = _check_pk_in_pre_segments(rowset_id, pre_segments, *key,
+                                                        dummy_version, delete_bitmap, &loc);
                     if (st.ok()) {
                         delete_bitmap->add({loc.rowset_id, loc.segment_id, dummy_version.first},
```

Comment:
Don't need this code, because already added in `_check_pk_in_pre_segments`.

Reply:
en, i fix _check_pk_in_pre_segments